pages/services/edge-lb/0.1.9/installing/index.md

```diff
@@ -85,18 +85,13 @@ Use the following CLI commands to provision the Edge-LB service account with the
 
 All CLI commands can also be executed via the [IAM API](/1.10/security/iam-api/).
```

Comment:
Hardcoded link to a dcos version.  How can we make this link non-dcos version specific (so that it still applies once 1.11 comes out).